components/monitoring/base/kustomization.yaml

```diff
@@ -0,0 +1,22 @@
+apiVersion: kustomize.config.k8s.io/v1beta1
+kind: Kustomization
+resources:
+  - install_prometheus.yaml
+  - prometheus-view.yaml
+
+namespace: "openshift-user-workload-monitoring"
+patches:
+  - target:
```

Comment:
why to patch local files? Why not to modify them directly?

Reply:
I had to remove these patches to be able to run this.

Reply:
I wanted to patch them through kustomization.yaml to make sure, in future if we plan to change the namespace we dont need change it every where. We are planning to add a lot of configs here for single change updating all the files does not make much sense to me.

Reply:
if we wanted to change the namespace name in the future, then it would make sense to change it in the local files as well - having something different in the patches and in the local files doesn't make any sense and it just brings confusion. 
Since the files are located in the same folder, in the same repository, and without having additional overlays, then using patches doesn't have any reasoning.

Reply:
ok, agree with the confusion part. but we need to update in so many places. It may be one time and never happen also. removing it.